src/ProgressOnderwijsUtils/SchemaReflection/DbColumnMetaData.cs

```diff
@@ -67,6 +77,7 @@ static ParameterizedSql BaseQuery(ParameterizedSql database)
                     DbObjectId = c.object_id
                     , ColumnName = c.name
                     , ColumnId = c.column_id
+                    , Ordinal = convert(int, row_number() over(partition by c.object_id order by c.column_id)) - 1

```

Comment:
Deze code is een beetje perf gevoelig, dus benchmark dit even.  Hoe dan ook kun je dit ook prima achteraf in C# in een for-lus doen, dus het hoeft niet zo.

Reply:
(Dit soort dingen zitten op de hot-path van startup, en dat is vooral vervelend tijdens ontwikkeling.